app/javascript/widget/components/ChatFooter.vue

```diff
@@ -58,9 +58,9 @@ export default {
       return getContrastingTextColor(this.widgetColor);
     },
     hideReplyBox() {
-      const { csatSurveyEnabled } = window.chatwootWebChannel;
+      const { allowMessagesAfterResolved } = window.chatwootWebChannel;
       const { status } = this.conversationAttributes;
-      return csatSurveyEnabled && status === 'resolved';
+      return allowMessagesAfterResolved && status === 'resolved';
```

Comment:
This method is hideReplyBox, right? 

The case should be `!allowMessagesAfterResolved and status === 'resolved'`, right?

Reply:
Done